openvdb/math/Vec3.h

```diff
@@ -71,9 +71,9 @@ class Vec3: public Tuple<3, T>
     template <typename Source>
     Vec3(Source *a)
     {
-        this->mm[0] = a[0];
-        this->mm[1] = a[1];
-        this->mm[2] = a[2];
+        this->mm[0] = static_cast<T>(a[0]);
```

Comment:
This change should be made in `Vec2.h` and `Vec4.h` as well.

Reply:
Done, and `Mat3/Mat4.h` too